ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -937,6 +937,7 @@ def verify_ocs_csv(ocs_registry_image=None):
             )
 
 
+@retry(AssertionError, 15, 10, 0)
```

Comment:
```suggestion
@retry(AssertionError, 15, 10, 1)
```
if backoff is 0, then delay will be 0 from 2nd iteration

Reply:
Good catch, fixed